src/main/webapp/i18n/de/featureOverview.json

```diff
@@ -109,14 +109,14 @@
                 },
                 "userInterface": {
                     "title": "Intuitive UI",
-                    "shortDescription": "Navigiere mit Hilfe der intuitiven Benutzeroberfläche durch die Klausur.",
-                    "descriptionTextOne": "Du führst die Klausur mit der Benutzeroberfläche des Artemis-Klausurmodus durch. Das Interface verbirgt alle anderen Funktionen von Artemis, um Ablenkungen zu vermeiden, und ermöglicht es dir, dich auf die Klausur selbst zu konzentrieren. Am oberen Rand zeigt eine Statusleiste den Status jeder Aufgabe an und verfolgt deinen Fortschritt bei jeder Aufgabe anhand einer Farbcodierung. Wenn du zum Beispiel ungespeicherte Änderungen vorgenommen hast, wird die jeweilige Aufgabe in der Statusleiste gelb hervorgehoben. Sobald du deinen Fortschritt speicherst oder er automatisch synchronisiert wird, wird er entsprechend angezeigt. Du kannst die Statusleiste auch benutzen, um zwischen den Aufgabe zu navigieren, indem du einfach auf die Aufgabe klickst.",
+                    "shortDescription": "Navigiere mit Hilfe der intuitiven Bedienoberfläche durch die Klausur.",
+                    "descriptionTextOne": "Du führst die Klausur mit der Bedienoberfläche des Artemis-Klausurmodus durch. Das Interface verbirgt alle anderen Funktionen von Artemis, um Ablenkungen zu vermeiden, und ermöglicht es dir, dich auf die Klausur selbst zu konzentrieren. Am oberen Rand zeigt eine Statusleiste den Status jeder Aufgabe an und verfolgt deinen Fortschritt bei jeder Aufgabe anhand einer Farbcodierung. Wenn du zum Beispiel ungespeicherte Änderungen vorgenommen hast, wird die jeweilige Aufgabe in der Statusleiste gelb hervorgehoben. Sobald du deinen Fortschritt speicherst oder er automatisch synchronisiert wird, wird er entsprechend angezeigt. Du kannst die Statusleiste auch benutzen, um zwischen den Aufgabe zu navigieren, indem du einfach auf die Aufgabe klickst.",
```

Comment:
I'm not sure about this change. Is it necessary? As far as I know there isn't a "Benutzerinoberfläche" and "Benutzeroberfläche" would just be the direct translation of "user interface". It already seems neutral to me, as it is the actual term translation.

Reply:
Yes, as the word does not exist according to duden, I will change it to Benutzeroberfläche again.